app/controllers/schools_controller.rb

```diff
@@ -1,14 +1,18 @@
+require 'byebug'
+
 class SchoolsController < ApplicationController
+  rescue_from ActiveRecord::MultiparameterAssignmentErrors,
```

Comment:
@ethanmills @kicferk1 I just decided to handle the exception. It looks less messy than the controller validations in my view

Reply:
What happens if we have two dates? This is a pattern we will need to use a lot I think

Reply:
We can worry about that when we get to it. We may get there and there are a few potential solutions.